antwar.config.js

```diff
@@ -0,0 +1,231 @@
+var path = require('path');
+var prevnextPlugin = require('antwar-prevnext-plugin');
+var markdown = require('./utilities/markdown');
+var highlight = require('./utilities/highlight');
+
+module.exports = {
+  // TODO: push to antwar.webpack.js or so. easier to handle ExtractTextPlugin etc.
+  webpack: {
+    common: function(options) {
+      var includes = options.includes;
+
+      return {
+        resolve: {
+          extensions: ['', '.js', 'jsx', '.scss'],
+          alias: {
+            Components: path.resolve('./components'),
+            Utilities: path.resolve('./utilities')
+          }
+        },
+        module: {
+          loaders: [
+            {
+              test: /\.jsx?$/,
+              loader: 'babel',
+              query: {
+                cacheDirectory: true,
+                compact: true,
+                presets: [
+                  require.resolve('babel-preset-es2015'),
+                  require.resolve('babel-preset-react')
+                ]
+              },
+              include: includes.concat([
+                path.dirname(require.resolve('antwar-helpers/components')),
+                path.dirname(require.resolve('antwar-helpers/layouts')),
+                path.join(__dirname, 'components'),
+                path.join(__dirname, 'layouts')
+              ])
+            },
+            // This is a weird one. Acorn dep?
+            {
+              test: /\.json$/,
+              loader: 'json',
+              include: [
+                require.resolve('globals/globals.json'),
+                require.resolve('regexpu-core/data/iu-mappings.json')
+              ]
+            }
+          ]
+        }
+      };
+    },
+    build: function(plugins) {
+      var ExtractTextPlugin = plugins.ExtractTextPlugin;
```

Comment:
Yeah. I'll push the webpack specific configuration to a file of its own. Now it's getting bundled for legacy reasons but splitting it up will allow me to remove extract-text-plugin from antwar core and keep this cleaner.

Basically I'm going to maintain only minimal configuration at antwar in its next version and push the configuration problem to a package of its own I can use across projects.